app/Models/Alertgroup.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace App;
+
+use App\Contracts\DBTable;
+use App\Notifications\UserWasRegistered as RegisteredNotification;
+use App\Notifications\PasswordWillReset as PasswordResetNotification;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Notifications\Notifiable;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+class Alertgroup extends Model
+{
+
+    /**
+     * @var string
+     */
+    protected $table = DBTable::ALERTGROUP;
+
+    /**
+     * @var string
+     */
+    protected $primaryKey = 'id';
+
+    /**
```

Comment:
@khanhpvrk Because your PK is not AI (auto increment), so you need to declare `$incrementing = false;`